docs/development/design/telemetry.rst

```diff
@@ -0,0 +1,232 @@
+Collect Data About Builds
+=========================
+
+We may want to take some decisions in the future about deprecations and supported versions.
+Right now we don't have data about the usage of packages and their versions on Read the Docs
```

Comment:
Moarrrrrrrrrr data! 🤓

Reply:
Now, more seriously: aren't we storing the build logs already somewhere? I think this proposal is more about storing **structured** data.

Reply:
build logs don't include the output from `pip list` or `conda list`

Reply:
but yeah, we already save the build output in json files in s3